data/theme.toml

```diff
@@ -5,8 +5,9 @@ facebook = 'https://facebook.com/'
 github = 'https://github.com/'
 instagram = 'https://instagram.com/'
 linkedin = 'https://linkedin.com/in/'
+mastodon = 'https://'
```

Comment:
Is this url correct?

Reply:
Yes, it is a prefix that is prepended to whichever account is configured.

Mastodon is federated, which means that there is no single domain. Instead, users create an account on an instance and communicate with the rest of the network from there.